ui/job-view/pushes/JobGroup.jsx

```diff
@@ -137,6 +138,15 @@ export class JobGroupComponent extends React.Component {
             tier={groupTier}
             toggleExpanded={this.toggleExpanded}
           />
+          {runnableVisible && expanded && (
+            <button
+              className="btn group-select-all-runnable"
+              type="button"
+              title="Select/deselect all jobs in this group"
```

Comment:
Where's the onClick event handler?

Reply:
I'm not using onclick because the existing system around PushJobs doesn't use it.
if onclick is preferred, I can switch to it.

Reply:
onClick would be preferred, thanks.